modules/infrastructure-moving-router.adoc

```diff
@@ -81,9 +75,9 @@ Update the `nodeSelector` stanza to reference the `infra` label as shown:
 ----
 $ oc get pod -n openshift-ingress
```

Comment:
Just adding `-o wide` to this command or cannot see those fields like `IP`, `NODE`. Since following steps need to ensure the pod is running on the `infra` node, so we should keep the old output.